pyoptsparse/pySNOPT/pySNOPT.py

```diff
@@ -627,43 +624,49 @@ def _getPenaltyParam(self, iw, rw):
         return xPen
 
     # fmt: off
-    def _snstop(self, ktcond, mjrprtlvl, minimize, n, nncon, nnobj, ns, itn, nmajor, nminor, nswap, condzhz, iobj, scaleobj,
-                objadd, fobj, fmerit, penparm, step, primalinf, dualinf, maxvi, maxvirel, hs, locj, indj, jcol, scales, bl, bu, fx, fcon, gcon, gobj, ycon,
-                pi, rc, rg, x, cu, iu, ru, cw, iw, rw):
+    def _snstop(self, ktcond, mjrprtlvl, minimize, n, nncon, nnobj, ns, itn, nmajor, nminor, nswap, condzhz, iobj,
+                scaleobj, objadd, fobj, fmerit, penparm, step, primalinf, dualinf, maxvi, maxvirel, hs, locj, indj,
+                jcol, scales, bl, bu, fx, fcon, gcon, gobj, ycon, pi, rc, rg, x, cu, iu, ru, cw, iw, rw):
         # fmt: on
         """
         This routine is called every major iteration in SNOPT, after solving QP but before line search
         We use it to determine the correct major iteration counting, and save some parameters in the history file.
-        If 'snSTOP function handle' is set to a function handle, then the callback is performed at the end of this function.
+        If 'snSTOP function handle' is set to a function handle, then it is called at the end of this function.
 
-        returning with iabort != 0 will terminate SNOPT immediately
+        Returns
+        -------
+        iabort : int
+            The return code expected by SNOPT. A non-zero value will terminate SNOPT immediately.
         """
         iterDict = {
             "isMajor": True,
             "nMajor": nmajor,
```

Comment:
One other thing I want to discuss is the name of these variables. Some match the SNOPT print out but others (e.g. `nMajor`) do not. Do we want to make things consistent in some way? It will break backwards compatibility if your post-processing script is explicitly looking for these names though.

Reply:
I would be fine with changing these to match SNOPT

Reply:
Hmm it's a little tricky though, because IMO SNOPT terminology is a little strange:
* `Feasible`/`Optimal` are adjectives being used as nouns
* Major but then Minors (I guess Major is always incremented by at most 1 but still)
* `Itns` will cause confusion with the existing key `iteration` which stores the callCounter so I've opted to skip this one

I guess the only ones I'm unhappy about in the current naming are `nMajor` and `nMinor` vs say `major` and `minor`, the others I'm actually okay with.

Reply:
I'm actually fine with the current names in order to not break backwards compatibility, we can revisit this later if needed.